mlflow/server/auth/basic_auth.ini

```diff
@@ -1,3 +1,5 @@
 [mlflow]
 default_permission = READ
 database_uri = sqlite:///basic_auth.db
+root_username = root
```

Comment:
```suggestion
admin_username = admin
```

@gabrielfu Can we rename this to `admin_username`? Having both `root` and `admin` might be confusing.

Reply:
sure